rules/integrations/aws/ml_cloudtrail_rare_method_by_city.toml

```diff
@@ -24,20 +24,83 @@ interval = "15m"
+understanding what is considered normal behavior within an organization, you can spot suspicious or malicious activity
+when deviations occur.
+
+This rule uses a machine learning job to detect an AWS API command that while not inherently suspicious or abnormal, is
+sourcing from a geolocation (city) that is unusual for the command. This can be the result of compromised credentials or
+keys used by a threat actor in a different geography than the authorized user(s).
+
+Detection alerts from this rule indicate an AWS API command or method call that is rare and unusual for the geolocation
+of the source IP address.
+
+#### Possible investigation steps
+
+- Identify the user account involved and the action performed. Verify whether it should perform this kind of action.
+    - Examine the user identity in the `aws.cloudtrail.user_identity.arn` field and the access key ID in the 
+    `aws.cloudtrail.user_identity.access_key_id` field, which can help identify the precise user context.
+    - The user agent details in the `user_agent.original` field may also indicate what kind of a client made the request.
+- Investigate other alerts associated with the user account during the past 48 hours.
+- Validate the activity is not related to planned patches, updates, or network administrator activity.
+- Examine the request parameters. These may provide indications as to the source of the program or the nature of the
+tasks it is performing.
+- Considering the source IP address and geolocation for the calling user who issued the command:
+    - Do they look normal for the calling user?
+    - If the source is an EC2 IP address, is it associated with an EC2 instance in one of your accounts, or could
+    it be sourcing from an EC2 instance that's not under your control?
+    - If it is an authorized EC2 instance, is the activity associated with normal behavior for the instance role or roles?
+    Are there any other alerts or signs of suspicious activity involving this instance?
+- Consider the time of day. If the user is a human, not a program or script, did the activity take place during a normal
+time of day?
+- Contact the account owner and confirm whether they are aware of this activity if suspicious.
+- If the account is suspected to have been compromised, track access to servers, services, and data accessed by the
+account in the last 24 hours to scope potentially compromised assets.
+
+### False positive analysis
 
-## Triage and analysis
+- False positives can occur if activity is coming from new employees based in a city with no previous history in AWS.
+- Examine the history of the command. If the command, which is visible in the `event.action field`, only manifested
+recently, it might be part of a new automation module or script. If it has a consistent cadence (for example, if it
+appears in small numbers on a weekly or monthly cadence), it might be part of a housekeeping or maintenance process.
```

Comment:
```suggestion
appears in small numbers on a weekly or monthly cadence), it might be part of a housekeeping or maintenance process. You can find the command in the `event.action field` field.
```